x/bank/client/rest/grpc_query_test.go

```diff
@@ -0,0 +1,71 @@
+package rest_test
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	grpctypes "github.com/cosmos/cosmos-sdk/types/grpc"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/cosmos/cosmos-sdk/x/bank/types"
+)
+
+func (s *IntegrationTestSuite) TestTotalSupplyGRPCHandler() {
+	val := s.network.Validators[0]
+	baseURL := val.APIAddress
+
+	testCases := []struct {
+		name     string
+		url      string
+		headers  map[string]string
+		respType fmt.Stringer
+		expected fmt.Stringer
+	}{
+		{
+			"test GRPC total supply",
+			fmt.Sprintf("%s/cosmos/bank/v1beta1/supply?height=1", baseURL),
```

Comment:
Just so that it's not confusing for other people
```suggestion
			fmt.Sprintf("%s/cosmos/bank/v1beta1/supply", baseURL),
```